canonical/snapcraft.io
# chore: Move listing page into publisher app

## Done
Moved the listing page into the publisher app.

The majority of this work is moving and renaming files, whilst also updating routes, renaming variables and probs so the components work as expected. There are no changes to functionality.

## How to QA
- Go to https://snapcraft-io-4865.demos.haus/<SNAP_NAME>/listing
- Check that it is in the application layout
- Check that you can still save changes

## Testing

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-14633
